tests/template_test.go

```diff
@@ -67,11 +72,13 @@ func TestTemplates(t *testing.T) {
 	templates, err := repo.Templates()
 	assert.NoError(t, err)
 
+	whiteListedTests := strings.Split(testPrefixWhiteList, ",")
+
 	for _, template := range templates {
 		templateName := template.Name
 		t.Run(templateName, func(t *testing.T) {
 			t.Parallel()
-			if !strings.Contains(templateName, "aws") || strings.Contains(templateName, "go") {
+			if !isWhiteListedTest(templateName, whiteListedTests) || strings.Contains(templateName, "go") {
```

Comment:
Your "rule engine" covers only 1 exclusion out of the current two. Can we come up with the one that would cover both needs?